pom.xml

```diff
@@ -1,7 +1,7 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.kohsuke</groupId>
-  <artifactId>github-api</artifactId>
+  <artifactId>github-api2</artifactId>
   <version>2.0-alpha1-SNAPSHOT</version>
```

Comment:
Putting these changes into a new artifact and designating `alpha1`.  No backward compatibility guarantees.